spec/support/authorize.rb

```diff
@@ -0,0 +1,3 @@
+def auth(user)
+  @request.env["HTTP_AUTHORIZATION"] = "Token token=#{user.auth_token}"
+end
```

Comment:
Yeah I'm not sure what happened with all the newlines, if there is a next time :) I'll be more careful.